### ISSUE TITLE
Error: OpenEye atom stereochemistry assumptions failed.

### ISSUE THREAD

**Issue by kadiliissaar** [openmm/openmmforcefields#141]
Hello!

I am receiving an error when working with molecules that include stereocentres, e.g.
```
smiles ='c1nc(c2c(n1)n(cn2)[C@H]3[C@@H]([C@@H]([C@H](O3)CO)O)O)N'
molecule = Molecule.from_smiles(smiles)
gaff.add_molecules(molecule)
```
**Exception:** Programming error: OpenEye atom stereochemistry assumptions failed.

(same happens with other molecules that have '@' in the smiles.)

I would be grateful for any insight you have on how to fix this.

Thank you for your help and best wishes,
Kadi

**Comment by kadiliissaar** [comment]
P.S. Also posted this issue to the openforcefield github
https://github.com/openforcefield/openforcefield/issues/742

**Comment by j-wags** [comment]
I've seen this before and suspect that it's an OpenFF dependency version issue -- So probably no action needed here.

**Comment by jchodera** [comment]
@j-wags : Should I pin `openmmforcefields` to require `openforcefield >=0.7.1`?

**Comment by j-wags** [comment]
I think it already is pinned to >=0.7.1:
https://github.com/omnia-md/conda-recipes/blob/master/openmmforcefields/meta.yaml#L72
https://anaconda.org/omnia/openmmforcefields/files

I'm guessing that this user had tried to update an old environment, leading to resolving on an old build of an otherwise-good version? Or something equivalent to force-installed deps?

**Comment by kadiliissaar** [comment]
I can confirm this got resolved when moving to openforcefield >=0.7.1 - thank you!
And yes, I was using an existing environment, that's probably why I didn't have the correct version of openforcefield.
